### PR TITLE
chore(cargo): Update dependency

### LINKED ISSUE #583: Groups - Typing indicator will persist (even when user is clicked outside of app)
**Issue:** In a group chat with User A,B, and C if User C sends a message and then clicks out of the app the typing indicator will persist for an extremely long time. Usually up to 5-6 minutes after the user sends the message 

**OS:**
-Linux
-MacOS
-Windows

**Screen recording:**

https://user-images.githubusercontent.com/93608357/229623052-433f8130-b29a-48dd-8fd4-448b55f51459.mov

### PR DESCRIPTION
**Pull request by dariusc93**
### What this PR does 📖

- Update dependency

### Which issue(s) this PR fixes 🔨

- *should* resolve #583

### CODE REVIEW

**Comment by dariusc93** [comment]
Branch can be merged after QA approval since the only change was upstream changes.

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**
